src/pages/billing/List.js

```diff
@@ -360,18 +359,33 @@ export const BillingList = ({ queryParams: { selectedName } }) => {
               p(['It may not exist, or you may not have access to it.'])
             ])
           ])],
-        [selectedName && hasBillingProjects, () => h(ProjectDetail, {
-          key: selectedName,
-          project: _.find({ projectName: selectedName }, billingProjects),
-          billingAccounts,
-          authorizeAndLoadAccounts
-        })],
-        [!_.isEmpty(projectsOwned) && !selectedName && hasBillingProjects, () => div(
-          { style: { margin: '1rem auto 0 auto' } }, [
+        [selectedName && _.some({ projectName: selectedName }, projectsOwned), () => {
+          const index = _.findIndex({ projectName: selectedName }, billingProjects)
+          return h(ProjectDetail, {
+            key: selectedName,
+            billingProject: billingProjects[index],
+            billingAccounts,
+            authorizeAndLoadAccounts,
+            updateBillingProject: _.flow(
+              withErrorReporting('Error updating billing project'),
+              Utils.withBusyState(setIsLoadingProjects)
+            )(async () => {
+              try {
+                const projects = billingProjects.slice()
+                projects[index] = await Ajax(signal).Billing.billingProject(selectedName)
+                setBillingProjects(projects)
+              } catch (_) {
+                await loadProjects()
+              }
```

Comment:
1. Could you rename the argument to `catch` to `e` for consistency?
2. I think you'll have to report the error yourself in this catch, since it'll prevent the error reporting wrapper from seeing it.

Reply:
for what it's worth - I'd be surprised if the linter doesn't complain about unused binds - _ is a convention for not caring about that particular value.
I'll add in the error reporting.